test/Libraries/RevitServicesTests/RevitServicesTests.cs

```diff
@@ -328,5 +329,34 @@ public void TwoInstancesOfSerializableIdAreEqual()
 
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void MaterialManagerCheckDefaultValues()
```

Comment:
probably not necessary, but it might be worth looking into if we can start the revit in localized german mode using RTF and test this stuff or, can the API switch the localization mode?